deploy/crs/v2/snippets/prior_version.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   annotations:
     consoleDesc: Use this snippet to deploy a prior product version
     consoleName: snippet-prior-version
-    consoleSnippet: "true"
+    consoleSnippet: true
```

Comment:
this change is unecessary

Reply:
oops, replacing back changed it by accident.